README.md

```diff
@@ -24,13 +26,14 @@ Then, to build the code, run:
 For tests, ensure you have a local Postgres instance running. While in the
 `terra-data-catalog` directory, initialize the database as follows:
 
-```
+```sh
 psql -f common/postgres-init.sql
 ```
 
 After the database is initialized, then you may run integration tests as follows:
 
-```
-render-configs.sh
-./gradlew :integration:runTest --scan --args="suites/FullIntegration.json /tmp/test"
+```sh
+./gradlew bootRun & # start up a local instance of the data catalog service
+render-configs.sh # render service account credentials needed for tests
+./gradlew :integration:runTest --args="suites/FullIntegration.json /tmp/test"
 ```
```

Comment:
Can we also add this to the readme?

```suggestion
```# Run Performance Tests:
```./render-configs.sh perf
```./gradlew :integration:runTest --args="suites/FullPerf.json /tmp/test"
```